cmd/clusterctl/client/config/providers_client.go

```diff
@@ -372,3 +380,37 @@ func validateProvider(r Provider) error {
 	}
 	return nil
 }
+
+func GetLastVersion(projectOwner string, projectName string) (string ) {
```

Comment:
Is there something different about outscale that it requires a different flow for getting the latest tag? Otherwise it's probably better to add the provider in the same way as the others and discuss / implement the change in versioning across all providers instead of just this one.

Reply:
I remove it.